infra/profile.sh

```diff
@@ -9,45 +9,41 @@ cleanup() {
 trap cleanup EXIT
 
 # TODO: take in file glob as command line argument
-PROFILES=tests/small/add.bril
+PROFILES=(tests/small/*.bril)
```

Comment:
This is fine for now, but we actually test on files outside the `small` folder as well. Can be fixed in a future PR or on this one.
The current `files.rs` testing harness actually just looks recursively for bril files, excluding ones in with `failing` or `should_fail` in the full path. Which is a hack

Reply:
Yeah, I'm not particularly happy with how we currently do it, we shouldn't need a high level language to decide whether we should exclude tests, it should be super easy to glob all our benchmarks.

I think we probably want to have actual benchmarks rather than tests or else we'll get a lot of pollution on our tables, which should help. We could put those in a bench/ folder or something